### PR TITLE
Add Query 'CloudFront Distribution Logging is Disabled' for AWS CloudFormation

### LINKED ISSUE #757: Add cloudfront_distribution_looging_disabled query for AWS CloudFormation
### Platform
AWS

### Provider
AWS

### Description
Ensure AWS CloudFront distribution with access logging is enabled

### PR DESCRIPTION
**Pull request by pedropereiraaa**
Closes #757

### CODE REVIEW

File: assets/queries/cloudFormation/logging_is_disabled/test/negative.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Resources:
+  myDistribution3:
+    Type: AWS::CloudFront::Distribution
+    Properties:
+      DistributionConfig:
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/logging_is_disabled/test/negative.yaml:6]: To make this sample valid. Add mandatory **DefaultCacheBehaviour** section inside **DistributionConfig** e.g:
```       
DefaultCacheBehavior:
    AllowedMethods:
    - GET
    - HEAD
    - OPTIONS
    TargetOriginId: myS3Origin
    ForwardedValues:
      QueryString: 'false'
      Cookies:
        Forward: none
    TrustedSigners:
    - 1234567890EX
    ViewerProtocolPolicy: allow-all
```

File: assets/queries/cloudFormation/logging_is_disabled/test/positive.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Resources:
+  myDistribution1:
+    Type: AWS::CloudFront::Distribution
+    Properties:
+      DistributionConfig:
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/logging_is_disabled/test/positive.yaml:6]: To make this sample valid. Add mandatory **DefaultCacheBehaviour** section inside **DistributionConfig** e.g:
```       
DefaultCacheBehavior:
    AllowedMethods:
    - GET
    - HEAD
    - OPTIONS
    TargetOriginId: myS3Origin
    ForwardedValues:
      QueryString: 'false'
      Cookies:
        Forward: none
    TrustedSigners:
    - 1234567890EX
    ViewerProtocolPolicy: allow-all
```

File: assets/queries/cloudFormation/logging_is_disabled/test/positive.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        Enabled: 'true'
+        Comment: Some comment
+        DefaultRootObject: index.html
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Resources:
+  myDistribution2:
+    Type: AWS::CloudFront::Distribution
+    Properties:
+      DistributionConfig:
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/logging_is_disabled/test/positive.yaml:21]: To make this sample valid. Add mandatory **DefaultCacheBehaviour** section inside **DistributionConfig** e.g:
```       
DefaultCacheBehavior:
    AllowedMethods:
    - GET
    - HEAD
    - OPTIONS
    TargetOriginId: myS3Origin
    ForwardedValues:
      QueryString: 'false'
      Cookies:
        Forward: none
    TrustedSigners:
    - 1234567890EX
    ViewerProtocolPolicy: allow-all
```

File: assets/queries/cloudFormation/logging_is_disabled/test/positive_expected_result.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+[
+	{
+		"queryName": "CloudFront Distribution Logging is Disabled",
+		"severity": "MEDIUM",
+		"line": 5
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/logging_is_disabled/test/positive_expected_result.json:5]: After adding **DefaultCacheBehavior** adjust the positive lines if necessary.

File: assets/queries/cloudFormation/logging_is_disabled/test/positive_expected_result.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+	},
+	{
+		"queryName": "CloudFront Distribution Logging is Disabled",
+		"severity": "MEDIUM",
+		"line": 32
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/logging_is_disabled/test/positive_expected_result.json:10]: After adding **DefaultCacheBehavior** adjust the positive lines if necessary.

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]
Requested some changes in the samples to make them valid.

**Review by ruigomescx — APPROVED** [review]

**Pull request merged**